### PR TITLE
feat(dapi): add `totalCreditsInPlatform` endpoint

### PR DESCRIPTION
**Pull request by owl352**
## Issue being fixed or feature implemented

Currently the getTotalCredits method cannot be called in js-dapi-client, moreover, even if it is prescribed, the implementation of the method call inside the module is not complete

## What was done?
The method has been added to PlatformPromiseClient and PlatformMethodsFacade. A number of changes were also made to the function names

## How Has This Been Tested?
Manual testing was performed on a local node built from source code

## Breaking Changes
No breaking changes

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [x] I have added "!" to the title and described breaking changes in the corresponding section if my code contains any
- [x] I have made corresponding changes to the documentation if needed

**For repository code-owners and collaborators only**
- [x] I have assigned this pull request to a milestone

### CODE REVIEW

File: packages/js-dapi-client/lib/methods/platform/PlatformMethodsFacade.js [hunk 1]
```diff
@@ -36,6 +37,7 @@ class PlatformMethodsFacade {
     this.getIdentityContractNonce = getIdentityContractNonceFactory(grpcTransport);
     this.getIdentityNonce = getIdentityNonceFactory(grpcTransport);
     this.getIdentityKeys = getIdentityKeysFactory(grpcTransport);
+    this.getTotalCredits = getTotalCreditsFactory(grpcTransport)
```
Comment (pshenmic) [inline packages/js-dapi-client/lib/methods/platform/PlatformMethodsFacade.js:40]: GetTotalCreditsInPlatform

File: packages/js-dapi-client/lib/methods/platform/getTotalCreditsOnPlatform/getTotalCreditsOnPlatformFactory.js [hunk 2]
```diff
@@ -61,7 +61,7 @@ function getTotalCreditsOnPlatformFactory(grpcTransport) {
     throw lastError;
   }
 
-  return getTotalCreditsOnPlatform;
+  return getTotalCreditsInPlatform;
```
Comment (pshenmic) [inline packages/js-dapi-client/lib/methods/platform/getTotalCreditsOnPlatform/getTotalCreditsOnPlatformFactory.js:64]: getTotalCreditsOnPlatform

**Review by pshenmic — CHANGES_REQUESTED** [review]

**Comment by shumkov** [comment]
@owl352 Thank you for your PR. Please fix conflicts and then we can run CI

**Review by pshenmic — APPROVED** [review]
Looks good to me

**Comment by pshenmic** [comment]
Closing in favor of https://github.com/dashpay/platform/pull/2059

**Pull request closed without merging**